docs/migration/4_0.md

```diff
@@ -783,6 +783,24 @@ The following properties where removed in 4.0:
 - Styles for following selectors `cx-close-account-modal`, `cx-close-account`, `cx-forgot-password`, `cx-register`, `cx-update-password-form`, `cx-user-form` were moved from `@spartacus/styles` to `@spartacus/user/profile/styles`.
 - Styles for following selectors `cx-login`, `cx-login-form` were moved from `@spartacus/styles` to `@spartacus/user/account/styles`.
 
+#### (a11y) Screen Reader Improvements: Headings
+
+- Most of these changes affect only headings on screen reader rotor feature. So changes are visible for sighted users. 
+- Store Names on Store finder list page changed to `h2` from a `div` 
```

Comment:
IMO we have separated part in documentation for component changes (like template), and fact that it's for a11y is less important than specification which component we edited. Please check other examples (e.g. CartListItemComponent)

I would see it more like
```
### StoreFinderListItemComponent
- `div[class.cx-store-name]` element has been changed to `h2[class.cx-store-name]` (a11y)

### StoreFinderStoresCountComponent
- ...
and so on
```